test/test_operations.py

```diff
@@ -991,6 +995,42 @@ def test_max_broadcast(self):
     xla_c = torch.max(xla_a, xla_b)
     self.assertEqual(c.data, xla_c.data.cpu())
 
+  def test_sgn(self):
+    xla_device = xm.xla_device()
+    t = torch.randn(2, 3, dtype=torch.cfloat)
+    t[0][0].real.div_(0)
+    t[0][0].imag.div_(0)
+    t[0][1] = 0
+    t[0][1].real.div_(0)
+    t[0][1].imag.div_(0)
+    t[1][0] = 0
+    t[1][1].real.div_(0)
+    t[1][1] = t[1][1].real.abs()
+    t[1][2].real.div_(0)
+    t[1][2] = t[1][1].real.abs() * -1
+    a = t.sgn()
+    xla_a = t.to(xla_device).sgn()
+    print("milad")
+    print(t)
+    print(a.data)
+    print(xla_a.data.cpu())
+    self.assertEqual(a.data, xla_a.data.cpu())
```

Comment:
Please add a comment regarding input, whether they are  `inf`, `-inf` or `nan`. This way it is easier for us to debug if this test failed.